src/views/Detail.vue

```diff
@@ -75,52 +76,63 @@
           v-for="(item, index) in items"
           :key="index"
         >
-          <ImageDetail
-            v-if="fileType === 'images'"
-            :file="item"
-          />
-          <DirectoryDetail
-            v-if="fileType === 'directories'"
-            :file="item"
-          />
-          <DocumentDetail
-            v-if="fileType === 'text'"
-            :file="item"
-          />
-          <AudioDetail
-            v-if="fileType === 'audio'"
-            :file="item"
-          />
-          <VideoDetail
-            v-if="fileType === 'video'"
+          <component
+            :is="componentType"
```

Comment:
I am not sure whether the reduction in LOC's here is worth the reduction in verbosity here. As a very experienced programmer, I find the component abstraction hard to understand (not to mention that it's hard to find the docs for it). 

I would suggest to keep this as it was (perhaps using else-if for increased efficiency). Alternatively, I can accept it as it is but please provide a reference (URL) to the `<component>`-component as it's very hard to Google.

Reply:
It is not so much LOC, nor efficiency, it just looks a bit unorganized, with a lot of imports that are only used sometimes. But I agree that this isn't an improvement, I was just trying to see if I could abstract it away nicely, but vue woulnd't let me. 

I reverted it. In the future the whole file can probably be turned into a detailBase component, and then the problem is solved. Really beyond the scope of this work pacakge.

Reply:
That sounds like a nice suggestion.

Reply:
(For the future.)